### PR TITLE
refactor cgcv.h: modernize a bit

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

File: src/backend/cgcv.h [hunk 1]
```diff
@@ -15,13 +15,16 @@
 #define CGCV_H
 //#pragma once
 
+struct Symbol;
+struct Classsym;
```
Comment (yebblies) [inline src/backend/cgcv.h]: Unrelated?

Comment (WalterBright) [inline src/backend/cgcv.h]: The types are used later in this file. Generally, I try to set things up so .h files don't have to have an order.

**Comment by yebblies** [comment]
Auto-merge toggled on
